examples/sdk/auth-direct/src/hooks/useAuth.tsx

```diff
@@ -70,8 +75,40 @@ export const AuthProvider = ({ children }: { children: ReactNode }): React.JSX.E
       .finally(() => setLoading(false))
   }
 
+  const createCred = async (code: string): Promise<CreateCredentialWithCodeResponse | undefined> => {
+    setLoading(true)
+    setError(undefined)
+
+    try {
+      const challenge = await dfnsApi().auth.createCredentialChallengeWithCode({
+        body: { code, credentialKind: 'Fido2' },
```

Comment:
do we have a `CredentialKind` type?

Reply:
in the newly generated SDK, these are string unions rather than enums, so there's no explicit `CredentialKind` enum exported from the `generated/` folder. (even though Intellisense still properly autocompletes & TS checks that this string is correct)